packages/std/src/traits.rs

```diff
@@ -164,6 +179,82 @@ pub trait Api {
         recovery_param: u8,
     ) -> Result<Vec<u8>, RecoverPubkeyError>;
 
+    #[allow(unused_variables)]
+    fn bls12_381_aggregate_g1(&self, g1s: &[u8]) -> Result<[u8; 48], VerificationError> {
+        // Support for BLS12-381 is added in 2.1, i.e. we can't add a compile time requirement for new function.
+        // Any implementation of the Api trait which does not implement this function but tries to call it will
+        // panic at runtime. We don't assume such cases exist.
+        // See also https://doc.rust-lang.org/cargo/reference/semver.html#trait-new-default-item
+        unimplemented!()
+    }
+
+    #[allow(unused_variables)]
+    fn bls12_381_aggregate_g2(&self, g2s: &[u8]) -> Result<[u8; 96], VerificationError> {
+        // Support for BLS12-381 is added in 2.1, i.e. we can't add a compile time requirement for new function.
+        // Any implementation of the Api trait which does not implement this function but tries to call it will
+        // panic at runtime. We don't assume such cases exist.
+        // See also https://doc.rust-lang.org/cargo/reference/semver.html#trait-new-default-item
+        unimplemented!()
+    }
+
+    #[allow(unused_variables)]
+    fn bls12_381_aggregate_pairing_equality(
+        &self,
+        ps: &[u8],
+        qs: &[u8],
+        r: &[u8],
+        s: &[u8],
+    ) -> Result<bool, VerificationError> {
+        // Support for BLS12-381 is added in 2.1, i.e. we can't add a compile time requirement for new function.
+        // Any implementation of the Api trait which does not implement this function but tries to call it will
+        // panic at runtime. We don't assume such cases exist.
+        // See also https://doc.rust-lang.org/cargo/reference/semver.html#trait-new-default-item
+        unimplemented!()
+    }
+
+    #[allow(unused_variables)]
+    fn bls12_381_hash_to_g1(
+        &self,
+        hash_function: HashFunction,
+        msg: &[u8],
+        dst: &[u8],
```

Comment:
Woudn't it be nicer to make this a string?

Reply:
What part exactly? The hash function or a different parameter?

Reply:
I was thinking of making it `dst: &str`. But then again in the  RFC 9380 it seems to be raw bytes, so better keep it as is. Closing here.